fuel-core/src/executor.rs

```diff
@@ -103,13 +104,15 @@ impl Executor {
         // setup and execute block
         let current_height = db.get_block_height()?.unwrap_or_default();
         let current_hash = db.get_block_id(current_height)?.unwrap_or_default();
+        let da_height = db.get_finalized_da_height().await.unwrap_or_default();
```

Comment:
Does default actually make sense here? Like this is saying that the da chain has really finalized block zero no matter what. Is that actually true?